lld/ELF/Arch/AArch64.cpp

```diff
@@ -1014,9 +1018,18 @@ AArch64BtiPac::AArch64BtiPac(Ctx &ctx) : AArch64(ctx) {
   // relocations.
   // The PAC PLT entries require dynamic loader support and this isn't known
   // from properties in the objects, so we use the command line flag.
-  pacEntry = ctx.arg.zPacPlt;
```

Comment:
It could be worth adding to the comment here. Something like:
```
By default we only use hint-space instructions, but if we detect the PAuthABI, which requires v8.3-A we can use the non-hint space instructions.
```

Reply:
Added, thanks, see b189f515cb02ad6b4aa78f6431ddae70c0838f3b